UnityProject/Assets/Scenes/ActiveScenes/Lobby.unity

```diff
@@ -353,16 +353,1776 @@ PrefabInstance:
       propertyPath: m_LocalEulerAnglesHint.z
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 252153304076729325, guid: 829f4ca992b848d6bb4d007fb9c112e1,
```

Comment:
What did you change in the lobby seen?

Reply:
something in the lobby was changed by the UI rounding script